indico/modules/events/abstracts/templates/management/_notification_tpl_list.html

```diff
@@ -1,86 +1,169 @@
+                                                    {%- trans %}in{% endtrans -%}
+                                                    <strong>
+                                                        {{ tracks[rule.track[0]].title }}
+                                                    </strong>
+                                                {% endif %}
+                                            {% endif %}
+                                            {% if 'contribution_type' in rule %}
+                                                    {% if not rule.contribution_type %}
+                                                        {%- trans -%}
+                                                            as <strong>no type</strong>
+                                                        {%- endtrans -%}
+                                                    {% elif rule.contribution_type[0] == '*' %}
+                                                        {%- trans -%}
+                                                            as <strong>any type</strong>
+                                                        {%- endtrans -%}
+                                                    {% else %}
+                                                        {%- trans %}as{% endtrans -%}
+                                                        <strong>
+                                                            {{ contrib_types[rule.contribution_type[0]].name }}
+                                                        </strong>
+                                                    {% endif %}
+                                            {% endif %}
+                                            {% if not loop.last %}
+                                                {%- trans %}or{% endtrans -%}
+                                            {% endif %}
+                                        </li>
+                                    {% endfor %}
                                 </ul>
-                                <a href="#" class="i-button js-delete-item icon-remove icon-only"
-                                   title="{% trans %}Delete template{% endtrans %}"
-                                   data-qtip-style="danger"
-                                   data-href="{{ url_for('.email_tpl_delete', tpl) }}"
-                                   data-method="DELETE"
-                                   data-confirm="{% trans %}Are you sure you want to delete this template?{% endtrans %}"
-                                   data-title="{% trans %}Delete template{% endtrans %}"
-                                   data-update="#email-template-manager">
-                                </a>
-                        </div>
-                    </div>
-                </li>
-            {% endfor %}
-        </ul>
+                                <div class="email-preview"
+                                     id="email-preview-{{ tpl.id }}"
```

Comment:
Looks like the whole tag would fit in one line without going over 120 chars width